src/lib/containers/EntityBadgeIcons.tsx

```diff
@@ -43,6 +43,7 @@ export const ENTITY_BADGE_ICONS_TOOLTIP_ID = 'EntityBadgeIconsTooltipID'
 
 export type EntityBadgeIconsProps = {
   entityId: string
+  versionNumber?: number
```

Comment:
We can now fetch the badge info for a specific entity version (useful for datasets).